spikeinterface/extractors/tests/common_tests.py

```diff
@@ -66,10 +66,17 @@ def test_open(self):
                 trace_scaled = rec.get_traces(segment_index=segment_index, return_scaled=True, end_frame=2)
                 assert trace_scaled.dtype == 'float32'
             
-            if isinstance(rec, NeoBaseRecordingExtractor):
-                # for neo based also test annotations propagation
-                rec = self.ExtractorClass(local_folder / path, all_annotations=True, **kwargs)
+                            
+    def test_neo_annotations(self):
+        for entity in self.entities:
 
+            if isinstance(entity, tuple):
+                path, kwargs = entity
+            elif isinstance(entity, str):
+                path = entity
+                kwargs = {}
+            if hasattr(self.ExtractorClass , "NeoRawIOClass"):
```

Comment:
I don't know if there is a better way of doing this. My other option was (example):
`self.ExtractorClass.__base__.__base__ == _NeoBaseExtractor`